src/fsharp/FSharp.Core/SR.fs

```diff
@@ -167,3 +163,4 @@ module internal SR =
     let GetString3(name:System.String, arg1:System.String, arg2:System.String, arg3:System.String) : System.String = 
         System.String.Format(resources.GetString(name, System.Globalization.CultureInfo.CurrentUICulture), arg1, arg2, arg3)
 
+    let GetResourceManager(_:int) = resources
```

Comment:
the `_:int` parameter is ugly, but i cannot write `GetResourceManager() = resources` because the compiler generate an error

```
FSC: errore FS0193: The module/namespace 'Microsoft.FSharp.Core' from compilation unit 'FSharp.Core' did not contain the namespace, module or type 'unit'
```
I think the compiler try to use `unit` for `()`, but unit is defined after, in `prim-types.fsi`